### PR TITLE
style(code-editor): Matching-braces Style Changed

### LINKED ISSUE #16363: [Beta] Matching Braces color in the editor needs to be changed
#### Issue Description
The matching braces color `#00ff00` in the beta looks weird and after some time it becomes difficult to trace the matching pair. So, with my idea of adding a border around the matching braces like in VSCode, will resolve this issue.
Please, have a look at the below GIF's in fullscreen by clicking on them to notice the difference effectively. **This issue will be addressed in my up-coming PR.**  

Default Setup:
before

After applying the proposed changes: 
after

#### Browser Information
* Browser Name, Version: Chrome(latest release) & Firefox(latest release).
* Operating System: Windows 10 Pro 32 Bit. 
* Mobile, Desktop, or Tablet: Desktop

@QuincyLarson, @raisedadead Thoughts on this?

### PR DESCRIPTION
**Pull request by vkWeb**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [ ] Tested changes locally. 
**Reason**: Code-mirror doesn't seem to work locally. 
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16363

#### Description
Added a border around the pair of matching braces like in VSCode and also It's uniform with the night mode. 

Default Setup:

After applying the changes:

### CODE REVIEW

File: client/less/code-mirror.less [hunk 1]
```diff
@@ -34,4 +34,9 @@
 
 .CodeMirror-lint-tooltip {
   z-index: 9999;
-}
\ No newline at end of file
+}
+
+div.CodeMirror span.CodeMirror-matchingbracket {
+  border: 0.5px solid #939393 !important;
+  color: #000;
```
Comment (Bouncey) [inline client/less/code-mirror.less]: This `color` doesn't work in night mode, the bracket completely disappears.

Perhaps we could make it `#939393` also, so the user knows what type of bracket is highlighted.

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: client/less/code-mirror.less (hunk 1 above)
Comment (vkWeb) [reply client/less/code-mirror.less]: @Bouncey I think `#939393` should be set as the color for brackets only in night mode and `#000000` in the normal state. 
What do you think?

**Comment by raisedadead** [comment]
Hi @vkWeb would you need any help with this, there is a conflict that needs to be resolved, and the review comment to be addressed.

Let us know.

**Comment by vkWeb** [comment]
@raisedadead Yes, I need help with this merge conflict. How can I resolve this? And Yes, I'll make the requested changes. Actually, my local environment is down, it's showing that `plugin not found` I'll re-clone my fork and will install it all over again. 
Sorry for the delay.

**Comment by raisedadead** [comment]
You start with rebasing your branch, same as current staging. Follow the rebasing instructions in the docs, except use it on your pr branch.

When you get to the conflict, resolve them in your editor. Add your changes, make sure the old change is not overwritten by yours.

And push back on this PR branch `git push --force`.

However,
A simpler way would be close this PR and start fresh of staging, after you refresh your local, since you are cloning anyways again and you don't have much code change.

Reachout to the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors) for help.

**Comment by vkWeb** [comment]
@raisedadead Thanks for the help - Okay, I'm closing this PR. I'll submit a new one within few days.

**Pull request closed without merging**